Battle.js

```diff
@@ -53,8 +96,9 @@ function checkWinner(roomID, roomData, socket, io) {
 
     // Draw card for player if any is left and their played cards does not exceed the maxDeckSize for the room.
     let player = Rooms.get(roomID).players.get(socket.id);
-    if (player.deck.cards.length > player.usedCards.length + player.hand.length && roomData.maxDeckSize > player.usedCards.length + player.hand.length) {
-        let pickedCard = drawCard(player.usedCards, player.deck.cards.length, player.hand);
+    if (player.deck.cards.length > player.usedCards.length + player.hand.length) {
```

Comment:
Why det her fjernet fix pls

```suggestion
    if (player.deck.cards.length > player.usedCards.length + player.hand.length && roomData.maxDeckSize > player.usedCards.length + player.hand.length) {
```